### DISCUSSION TITLE
How can I scan without ping，like nmap -Pn

### DISCUSSION THREAD

**Discussion by Ma1Dong** [projectdiscovery/naabu#443] [Q&A] [answered]
thinks!

**Comment by olearycrew** [comment, marked as answer]
The same flag - `-Pn` is available for Naabu.

Thanks!

![CleanShot 2023-04-19 at 12 28 16](https://user-images.githubusercontent.com/6044920/233139959-32341431-7293-49d8-b9e8-db509d1455f9.png)
